accname/index.html

```diff
@@ -554,9 +554,8 @@ <h4>Computation steps</h4>
                   </div>
                 </li>
                 <li id="comp_embedded_control">
-                  <span id="step2C"><!-- Don't link to this legacy numbered ID. --></span><em>Embedded Control:</em> Otherwise, if the <code>current node</code> is a control embedded within the label
-                  (e.g. any element directly referenced by <code>aria-labelledby</code>) for another <a class="termref">widget</a>, where the user can adjust the embedded control's value, then return
-                  the embedded control as part of the text alternative in the following manner:
+                  <span id="step2C"><!-- Don't link to this legacy numbered ID. --></span><em>Embedded Control:</em> Otherwise, if the <code>current node</code> is a user-adjustable control embedded
```

Comment:
"user-adjustable" doesn't account for an embedded `readonly` field. 

Do we care about this?
```html
<label for=""cb>
  <input type="cb" name="cb" id="id">
  Flash the screen <input readonly value="3"> times.
</label>
```

Will the new language, the (admittedly contrived) label above would be "  Flash the screen times."

Reply:
Actually I guess the old text doesn't account for that either. "user-perceivable" perhaps? Or just remove that adjective?

Reply:
"user-perceivable" perhaps? 

```suggestion
                  <span id="step2C"><!-- Don't link to this legacy numbered ID. --></span><em>Embedded Control:</em> Otherwise, if the <code>current node</code> is a user-perceivable control embedded
```

Or just remove that adjective?

Reply:
Note: This includes disabled fields too, but I think that's good. What do you all think about this (also contrived) example?

```html
<label for=""cb>
  <input type="cb" name="cb" id="id">
  Flash the screen <input disabled value="3"> times.
</label>
```